hcipy/optics/segmented_mirror.py

```diff
@@ -0,0 +1,237 @@
+import os
+import numpy as np
+import matplotlib.pyplot as plt
+
+from .optical_element import OpticalElement
+from ..aperture import circular_aperture, hexagonal_aperture, make_segmented_aperture
+from ..field import Field, make_hexagonal_grid, make_pupil_grid, evaluate_supersampled
+from ..io import write_fits
+from ..plotting import imshow_field
+
+
+pupil_size = 1024
+PUP_DIAMETER = 15.   # m
+
+
+def get_atlast_aperture(normalized=False, with_segment_gaps=True, segment_transmissions=1, write_to_disk=False, outDir=None):
```

Comment:
It would be great to refactor either the segmented aperture functions or the SegmentedMirror() class such that we can use the original aperture functions and don't need to create a duplicate just because we need to return the segment positions.

Reply:
Yes, this should be an optional parameter `return_segment_positions` for every segmented aperture, starting with `make_segmented_aperture()`.

Reply:
Should it really be `make_segmented_aperture()` that has a flag for returning the segment positions? The segment positions are an input to that function, so wherever it is used, they will already be available - would there still be a reason to have `make_segmented_aperture()` return the segment positions nevertheless?

Reply:
Agreed. I realized this as well, while letting make_atlast_aperture() return the segment positions. Ignore the last part of that sentence.

Reply:
OK. I added the `return_segment_positions` keyword for the HiCAT and LUVOIR aperture functions, they will come up in the next push.